### DISCUSSION TITLE
My CI broke since I introduced Sign in with Apple

### DISCUSSION THREAD

**Discussion by ElenaVielva** [fastlane/fastlane#16995] [Q&A]
I have Fastlane setup on the CI . It uploads to Testflight automatically on commits in the beta branch. However, since I introduced Sign in with Apple I get this error
```
❌  error: Provisioning profile "iOS Team Provisioning Profile: *****" doesn't support the Sign In with Apple capability. (in target '****' from project '****')
❌  error: Provisioning profile "iOS Team Provisioning Profile: ****" doesn't include the com.apple.developer.applesignin entitlement. (in target '****' from project '****')
```

I found this [issue](https://github.com/fastlane/fastlane/issues/15707), but the solution to force match didn't work for me. I am able, thought, to run the lane from my development machine (so I guess it gets some local configuration)

I am using Fastlane version 2.139.0

**Comment by skelet2394** [comment]
It seems that you have provisioning profiles issue
You need to update them on your CI to successfully build
https://codesigning.guide can help you in understanding problem and will give you some options to resolve issue
